101-function-app-storage-private-endpoints/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,210 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location to deploy resources to."
+      }
+    },
+    "isContributor": {
+      "type": "bool",
+      "defaultValue": true,
+      "metadata": {
+        "description": "If the deployment principal is not a contributor and doesn't have the necessary permissions to run a Microsoft.Resources/deploymentScripts script, set this to false so that the custom role will be created and assigned to the principal. "
```

Comment:
the deployment principal has to be at least contributor or the template deployment will fail...